jsonexporter/version.go

```diff
@@ -1,3 +1,3 @@
 package jsonexporter
 
-const Version = "0.0.1"
+const Version = "0.1.1"
```

Comment:
I don't have specific versioning policy for this kind of small project but would be better to bump this to `1.0.0` to indicate breaking change?